### PR TITLE
Add Kotlin Team section

### PR DESCRIPTION
**Pull request by Southpaw1496**
The first commit adds a Kotlin Team section and existing Quilt team-members with cards. New members who need new cards will be added later.

### CODE REVIEW

**Comment by Southpaw1496** [comment]
Potatoboy hasn't responded to my request for details for a card, so I'm just going to go ahead and we can add him later.

File: src/_about/teams.md [hunk 1]
```diff
@@ -211,6 +211,15 @@ tooling, file repositories and API services for modders. This team is responsibl
...
 </div>
 
+## Kotlin
+
+The Kotlin Team is responsible for maintaining the Quilt Kotlin Libraries (QKL), a set of libraries based off QSL and designed for use with the Kotlin programming language. These libraries provide an official way to use Kotlin on Quilt, as well as wrappers for QSL and the Minecraft codebase for easier use in Kotlin.
+
+<div class="developer team-grid">
+    {% include team/Octal.liquid %}
+    {% include team/SilverAndro.liquid %}
+</div>
```
Comment (Oliver-makes-code) [inline src/_about/teams.md:221]: We'd need cards for @NoComment1105 and @PotatoPresident, but otherwise it looks good

**Review by Oliver-makes-code — CHANGES_REQUESTED** [review]

File: src/_about/teams.md (hunk 1 above)
Comment (Southpaw1496) [reply src/_about/teams.md]: I've added NoComment, Potato will need a new card as he's not part of any other Quilt teams. I've sent him a message about it, but he's very busy at the moment, so I'm planning to add his card later if I don't hear from him before this is merged.

**Comment by Southpaw1496** [comment]
I think I'm responsible enough to merge without Gdude's approval, wouldn't want to interrupt his BlanketCon-ing.

*famous last words*

**Pull request merged**